website/notifications/model.py

```diff
@@ -4,18 +4,27 @@
 from modularodm.exceptions import ValidationValueError
 
 from website.project.model import Node
-from website.notifications.constants import NOTIFICATION_TYPES
+from website.notifications.constants import NOTIFICATION_TYPES, NODE_SUBSCRIPTIONS_AVAILABLE, USER_SUBSCRIPTIONS_AVAILABLE
+import re
 
 
 def validate_subscription_type(value):
     if value not in NOTIFICATION_TYPES:
         raise ValidationValueError
 
 
+def validate_event_type(value):
+    prefix = re.compile('[[a-zA-Z0-9]*_]?')
+    if prefix.search(value):
+        value = 'file_updated'
```

Comment:
Basically, whenever something having to do with a 'file_updated', it is preceded by a user ID.   So to fix that, I added a line of regex to allow for a user ID and an underscore.  The test that Travis used was "xyz42_file_updated".  Is there a more common way this has been handled in other OSF files?